### ISSUE TITLE
Unable to attain the correct value of error after training

### ISSUE THREAD

**Issue by Meekahz** [odysseaspap/msc_thesis#5]
I've trained the de calibrated data but the paper plots that was resulted was nothing close to the values on the readme. Could anyone advice.

**Comment by Meekahz** [comment]
Epoch 1/32
 - 116s - loss: 159.6101 - quat_loss: 0.0793 - cloud_loss: 159.6101 - quat_rot_angle_error: 9.0982 - quat_tilt_error: 5.1371 - quat_pan_error: 3.7989 - quat_roll_error: 5.1606 - val_loss: 156.8770 - val_quat_loss: 0.0798 - val_cloud_loss: 156.8770 - val_quat_rot_angle_error: 9.1459 - val_quat_tilt_error: 4.9509 - val_quat_pan_error: 4.0468 - val_quat_roll_error: 5.0790
Epoch 2/32
 - 111s - loss: 148.8755 - quat_loss: 0.0773 - cloud_loss: 148.8755 - quat_rot_angle_error: 8.8663 - quat_tilt_error: 4.9653 - quat_pan_error: 3.5909 - quat_roll_error: 4.9904 - val_loss: 156.5509 - val_quat_loss: 0.0796 - val_cloud_loss: 156.5509 - val_quat_rot_angle_error: 9.1277 - val_quat_tilt_error: 4.9470 - val_quat_pan_error: 4.0109 - val_quat_roll_error: 5.0791
Epoch 3/32
 - 112s - loss: 148.3191 - quat_loss: 0.0771 - cloud_loss: 148.3191 - quat_rot_angle_error: 8.8398 - quat_tilt_error: 4.9636 - quat_pan_error: 3.5340 - quat_roll_error: 4.9895 - val_loss: 156.4668 - val_quat_loss: 0.0796 - val_cloud_loss: 156.4668 - val_quat_rot_angle_error: 9.1233 - val_quat_tilt_error: 4.9441 - val_quat_pan_error: 3.9922 - val_quat_roll_error: 5.0799
Epoch 4/32
 - 111s - loss: 147.4520 - quat_loss: 0.0767 - cloud_loss: 147.4520 - quat_rot_angle_error: 8.7963 - quat_tilt_error: 4.9619 - quat_pan_error: 3.4430 - quat_roll_error: 4.9892 - val_loss: 156.6261 - val_quat_loss: 0.0796 - val_cloud_loss: 156.6261 - val_quat_rot_angle_error: 9.1234 - val_quat_tilt_error: 4.9430 - val_quat_pan_error: 3.9883 - val_quat_roll_error: 5.0795
Epoch 5/32
 - 112s - loss: 146.0896 - quat_loss: 0.0762 - cloud_loss: 146.0896 - quat_rot_angle_error: 8.7370 - quat_tilt_error: 4.9586 - quat_pan_error: 3.3130 - quat_roll_error: 4.9884 - val_loss: 157.0173 - val_quat_loss: 0.0797 - val_cloud_loss: 157.0173 - val_quat_rot_angle_error: 9.1371 - val_quat_tilt_error: 4.9482 - val_quat_pan_error: 4.0041 - val_quat_roll_error: 5.0796
Epoch 6/32
 - 112s - loss: 143.9810 - quat_loss: 0.0753 - cloud_loss: 143.9810 - quat_rot_angle_error: 8.6314 - quat_tilt_error: 4.9558 - quat_pan_error: 3.0921 - quat_roll_error: 4.9872 - val_loss: 159.1911 - val_quat_loss: 0.0804 - val_cloud_loss: 159.1911 - val_quat_rot_angle_error: 9.2175 - val_quat_tilt_error: 4.9505 - val_quat_pan_error: 4.1206 - val_quat_roll_error: 5.0816

Epoch 00006: ReduceLROnPlateau reducing learning rate to 0.0004000000189989805.
Epoch 7/32
 - 112s - loss: 139.8164 - quat_loss: 0.0734 - cloud_loss: 139.8164 - quat_rot_angle_error: 8.4123 - quat_tilt_error: 4.9520 - quat_pan_error: 2.6273 - quat_roll_error: 4.9846 - val_loss: 161.0045 - val_quat_loss: 0.0810 - val_cloud_loss: 161.0045 - val_quat_rot_angle_error: 9.2903 - val_quat_tilt_error: 4.9499 - val_quat_pan_error: 4.2326 - val_quat_roll_error: 5.0826
Epoch 8/32
 - 114s - loss: 138.0831 - quat_loss: 0.0726 - cloud_loss: 138.0831 - quat_rot_angle_error: 8.3188 - quat_tilt_error: 4.9504 - quat_pan_error: 2.4140 - quat_roll_error: 4.9827 - val_loss: 161.1151 - val_quat_loss: 0.0811 - val_cloud_loss: 161.1151 - val_quat_rot_angle_error: 9.2971 - val_quat_tilt_error: 4.9423 - val_quat_pan_error: 4.2460 - val_quat_roll_error: 5.0839
Restoring model weights from the end of the best epoch
Epoch 00008: early stopping
/home/moovita/anaconda3/envs/ah/
